hever/inc/wpcom-colors.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+// Custom Colors: Hever
+function hever_define_color_annotations( $colors ) {
+	// Background Color
+	// --wp--preset--color--background
+	add_color_rule(
+		'bg',
+		$colors['background'],
+		array(
+
+			// This placeholder is needed to make the color annotations work
+			array( '.wp--preset--color--background', 'background-color' ),
+
+		),
+		__( 'Background Color' )
```

Comment:
Is it expected not to have translation-text-domain on this and few other `__()`s in this file?